Gemfile.lock

```diff
@@ -79,17 +100,21 @@ PLATFORMS
 
 DEPENDENCIES
   activerecord (= 4.2.5)
+  hashie
   minitest
   pry
   rake
   require_all
+  sass
   shotgun
   sinatra
   sinatra-activerecord
   slim
+  sprockets
   sqlite3
   thin
+  tilt-jbuilder (>= 0.4.0)
   tux
 
 BUNDLED WITH
-   1.15.3
+   1.13.6
```

Comment:
Why did you downgrade bundler?

Reply:
I have no idea. I suppose there was a conflict with some gem or smth.